VmAgent.Core/MultiLogger.cs

```diff
@@ -1,15 +1,10 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
```

Comment:
we should leave the license as is, as this is OSS code

Reply:
I thought I had put it back. Done